client/src/App.vue

```diff
@@ -595,12 +595,17 @@ document.addEventListener('focusout', (event) => {
   someInputFocused = false;
 }, false);
 
+
 document.addEventListener('contextmenu', (event) => {
   if (someInputFocused) return;
-
+  let nearbySelect = false;
   if (isOverlay3D(event.target)) {
     getViewCoordinates(event, tmpVec2);
-    propsSelector.select(tmpVec2, main.propSettings.strafe);
+    // TODO Set Keybind for CTRL?
+    if (event?.ctrlKey && inputListener.getStrafeKey()) {
```

Comment:
`getStrafeKey()` does not tell you if the strafe key is pressed, it just returns the key value bound to the strafe action (should be Shift by default).

`runPressed()` and `strafePressed()` will tell you if Ctrl and Shift are pressed (assuming default bindings)